app/javascript/comments.vue

```diff
@@ -235,7 +235,14 @@ export default {
           this.tab = 'comment'
           this.buttonDisabled = false
           this.resizeTextarea()
-          this.toast('コメントを投稿しました！')
+          if (
+            this.currentUser.roles.includes('mentor') &&
+            this.commentableType === 'Product'
+          ) {
+            if (this.productCheckerId) {
+              this.toast('コメントを投稿しました！')
+            }
+          } else this.toast('コメントを投稿しました！')
```

Comment:
```suggestion
          if (
            this.currentUser.roles.includes('mentor') &&
            this.commentableType === 'Product'
          ) {
            if (this.productCheckerId) {
              this.toast('コメントを投稿しました！')
            }
          } else {
            this.toast('コメントを投稿しました！')
          }
```

ifの方が複数行でelseが1行になっているとわかりづらいのでこちらがいいかもです。